builtin/credential/radius/path_config.go

```diff
@@ -0,0 +1,218 @@
+package radius
+
+import (
+	"strings"
+
+	"github.com/fatih/structs"
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+)
+
+func pathConfig(b *backend) *framework.Path {
+	return &framework.Path{
+		Pattern: `config`,
+		Fields: map[string]*framework.FieldSchema{
+			"host": &framework.FieldSchema{
+				Type:        framework.TypeString,
+				Description: "radius host to connect",
+			},
+
+			"port": &framework.FieldSchema{
+				Type:        framework.TypeInt,
+				Default:     1812,
+				Description: "radius port (default: 1812)",
+			},
+			"secret": &framework.FieldSchema{
+				Type:        framework.TypeString,
+				Description: "secret shared with the radius server",
+			},
+			"allow_unknown_users": &framework.FieldSchema{
+				Type:        framework.TypeBool,
+				Default:     false,
+				Description: "enable granting default policy upon successful RADIUS authentication (default: false)",
+			},
+			"reauth_on_renew": &framework.FieldSchema{
```

Comment:
This shouldn't be optional. This is normal Vault behavior for backends that can perform this capability. The authentication should be attempted again, and if the resultant set of policies have changed, the renewal should be disallowed.